apps/vaults/components/details/tabs/VaultDetailsStrategies.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {useMemo} from 'react';
+import dynamic from 'next/dynamic';
+import IconCopy from '@yearn-finance/web-lib/icons/IconCopy';
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
+import {formatBN, formatToNormalizedValue} from '@yearn-finance/web-lib/utils/format.bigNumber';
+import {formatAmount} from '@yearn-finance/web-lib/utils/format.number';
+import {formatDuration} from '@yearn-finance/web-lib/utils/format.time';
+import {copyToClipboard, parseMarkdown} from '@yearn-finance/web-lib/utils/helpers';
+import IconChevron from '@common/icons/IconChevron';
+import {formatPercent} from '@common/utils';
+
+import type {LoaderComponent} from 'next/dynamic';
+import type {ReactElement} from 'react';
+import type {TYearnVault, TYearnVaultStrategy} from '@common/types/yearn';
+import type {TGraphForStrategyReportsProps} from '@vaults/components/graphs/GraphForStrategyReports';
+
+const GraphForStrategyReports = dynamic<TGraphForStrategyReportsProps>(async (): LoaderComponent<TGraphForStrategyReportsProps> => import('@vaults/components/graphs/GraphForStrategyReports'), {ssr: false});
+
+type TRiskScoreElementProps = {
+	label: string;
+	value: number;
+};
+function	RiskScoreElement({label, value}: TRiskScoreElementProps): ReactElement {
+	return (
+		<div className={'flex flex-row items-center justify-between'}>
+			<p className={'text-sm text-neutral-500'}>{label}</p>
+			<p className={'font-number text-sm text-neutral-900'}>{value}</p>
+		</div>
+	);
+}
+
+function	VaultDetailsStrategy({currentVault, strategy}: {currentVault: TYearnVault, strategy: TYearnVaultStrategy}): ReactElement {
+	const	riskScoreElementsMap = useMemo((): TRiskScoreElementProps[] => ([
+		{label: 'TVL Impact', value: strategy?.risk?.riskDetails?.TVLImpact},
+		{label: 'Audit Score', value: strategy?.risk?.riskDetails?.auditScore},
+		{label: 'Code Review Score', value: strategy?.risk?.riskDetails?.codeReviewScore},
+		{label: 'Complexity Score', value: strategy?.risk?.riskDetails?.complexityScore},
+		{label: 'Longevity Impact', value: strategy?.risk?.riskDetails?.longevityImpact},
+		{label: 'Protocol Safety Score', value: strategy?.risk?.riskDetails?.protocolSafetyScore},
+		{label: 'Team Knowledge Score', value: strategy?.risk?.riskDetails?.teamKnowledgeScore},
+		{label: 'Testing Score', value: strategy?.risk?.riskDetails?.testingScore}
+	]), [strategy]);
```

Comment:
nit: There's a lot of question marks and long value here, could we do something like:
```suggestion
	const	riskScoreElementsMap = useMemo((): TRiskScoreElementProps[] => {
		const {riskDetails} = strategy?.risk || {};
		return [
			{label: 'TVL Impact', value: riskDetails?.TVLImpact},
			{label: 'Audit Score', value: riskDetails?.auditScore},
			{label: 'Code Review Score', value: riskDetails?.codeReviewScore},
			{label: 'Complexity Score', value: riskDetails?.complexityScore},
			{label: 'Longevity Impact', value: riskDetails?.longevityImpact},
			{label: 'Protocol Safety Score', value: riskDetails?.protocolSafetyScore},
			{label: 'Team Knowledge Score', value: riskDetails?.teamKnowledgeScore},
			{label: 'Testing Score', value: riskDetails?.testingScore}
		];
	}, [strategy]);
```

Reply:
Also, a follow up on this, I'm not entirely sure why `riskScoreElementsMap` is using `useMemo`.

There's not much happening there :)

Reply:
> Also added `|| 0` just to be sure

In that case let's change the type
```
type TRiskScoreElementProps = {
	label: string;
	value?: number;
};
```
and then
```
{riskScoreElementsMap.map(({label, value}): ReactElement => (
	<RiskScoreElement
		key={label}
		label={label}
		value={value || 0} />
))}
```
so we only need one `|| 0`